packages/app/ios_config.sh

```diff
@@ -88,7 +88,10 @@ done
 
 if [[ ${_SEARCH_RESULT} ]]; then
   _JSON_OUTPUT_RAW=$(cat "${_SEARCH_RESULT}")
-  _RN_ROOT_EXISTS=$(ruby -Ku -e "require 'rubygems';require 'json'; output=JSON.parse('$_JSON_OUTPUT_RAW'); puts output[$_JSON_ROOT]" || echo '')
+  if ! _RN_ROOT_EXISTS_TMP=$(ruby -Ku -e "require 'json'; output=JSON.parse('$_JSON_OUTPUT_RAW'); puts output[$_JSON_ROOT]"); then
+    echo "error: Failed to parse firebase.json, check for syntax errors."
+  fi
+  # Check works but appears to nullify the json contents after.
```

Comment:
I think we want to fail fast on this, so toss an exit 1 in there and let it die

then no worries about nulled json contents after?

```suggestion
    echo "error: Failed to parse firebase.json, check for syntax errors." && exit 1
  fi
```